ports/stm32/powerctrl.c

```diff
@@ -926,8 +938,17 @@ void powerctrl_enter_standby_mode(void) {
     // Restore EWUP state
     PWR->CSR2 |= csr2_ewup;
     #elif defined(STM32H7)
-    EXTI_D1->PR1 = 0x3fffff;
-    PWR->WKUPCR |= PWR_WAKEUP_FLAG1 | PWR_WAKEUP_FLAG2 | PWR_WAKEUP_FLAG3 | PWR_WAKEUP_FLAG4 | PWR_WAKEUP_FLAG5 | PWR_WAKEUP_FLAG6;
+    // Clear and mask D1 EXTIs.
+    EXTI_D1->PR1 = 0x3fffffu;
+    EXTI_D1->IMR1 &= ~(0xFFFFu); // 16 lines
+
+    // Clear and mask D2 EXTIs.
+    EXTI_D2->PR1 = 0x3fffffu;
+    EXTI_D2->IMR1 &= ~(0xFFFFu); // 16 lines
+
+    // Disable all wakeup pins and clear flags.
+    PWR->WKUPEPR &= ~(0x1Fu);
+    PWR->WKUPCR |= PWR_WAKEUP_FLAG_ALL;
```

Comment:
What if you enabled some wake up pins in code, to wake the MCU?  Will that be disabled?

Reply:
I'm not sure, never tested standby with wakeup pins before, it's usually used with RTC.

Reply:
In that case I think it's best not to disable these wakeup pins.

Reply:
Okay.